src/Variable.cpp

```diff
@@ -849,12 +849,12 @@ void Variable::SetFillMode(v8::Local<v8::String> property, v8::Local<v8::Value>
     v8::Isolate* isolate = info.GetIsolate();
     Variable* obj = node::ObjectWrap::Unwrap<Variable>(info.Holder());
     if (!val->IsBoolean()) {
-        isolate->ThrowException(v8::Exception::TypeError(v8::String::NewFromUtf8(isolate, "Expecting a boolean")));
+        isolate->ThrowException(v8::Exception::TypeError(v8::String::NewFromUtf8(isolate, "Expecting a boolean", v8::NewStringType::kNormal).ToLocalChecked()));
         return;
     }
-    int v = val->BooleanValue() ? 1 : 0;
+    int v = val->BooleanValue(isolate) ? 1 : 0;
```

Comment:
This is the only change that is not compatible with node <= 11.
Could you wrap this call in a `#if` directive, so that we kept the old version for node <= 11?

Reply:
you can check NODE_MAJOR_VERSION to tell

Reply:
Done, let's see if I got it right...

Reply:
Thank you, seems fine. Beware that there are two more usage of BooleanValue in Variable.cpp. Could you fix that ones too?

Reply:
I setup actions for node >= 8, you can see CI results of your further commits in action tab, e.g. your last commit: https://github.com/parro-it/netcdf4/actions/runs/79261355

The workflow cannot run on node < 8 baecause mocha does not support it anymore...

Reply:
Done all instances of BooleanValue.

Reply:
Awesome! CI is passing now...

Reply:
Going to merge the change

Reply:
the PR, sorry